server/src/services/applicant.service.ts

```diff
@@ -0,0 +1,36 @@
+import { ILogger } from '../logger';
+import { Resume } from '../models';
+import { FeedbackRepository } from '../repositories/feedback';
+import { ResumeRepository } from '../repositories/resume';
+import { StudentRepository } from '../repositories/student';
```

Comment:
```suggestion
import { FeedbackRepository } from '../repositories/feedback.repository';
import { ResumeRepository } from '../repositories/resume.repository';
import { StudentRepository } from '../repositories/student.repository';
```
Also need to rename `resume.ts` to `resume.repository.ts` for consistency